### PR TITLE
Property is the default cid

### PR DESCRIPTION
**Pull request by GCorbel**
In order to have a `for` attribute for `label` tag and an id for inputs, I added a default cid value.

### CODE REVIEW

**Comment by GCorbel** [comment]
Ping @spruce , I can merge it myself if you want.

File: addon/components/form-group.js [hunk 1]
```diff
@@ -3,5 +3,16 @@ import layout from '../templates/components/form-group';
...
   layout: layout,
-  tagName: ''
+  tagName: '',
+  property: null,
+  id: Ember.computed('cid', 'property', {
+    get: function() {
+      if (this.get('cid')) {
+        return this.get('cid');
+      } else {
+        console.log(this.get('property'));
```
Comment (pedrokiefer) [inline addon/components/form-group.js]: Any need to be logging this?

File: tests/unit/components/em-checkbox-test.js [hunk 2]
```diff
@@ -30,3 +30,11 @@ test('cid correctly sets the id for the checkbox and it\'s label', function(asse
   assert.equal(this.$(':checkbox').attr('id'), 'test-cid', 'checkbox has correct id');
   assert.equal(this.$('label').attr('for'), 'test-cid', 'label has correct \'for\'');
 });
+
+test('cid it property by default', function(assert) {
```
Comment (pedrokiefer) [inline tests/unit/components/em-checkbox-test.js]: typo, should be "is property"

File: tests/unit/components/em-select-test.js [hunk 3]
```diff
@@ -138,6 +138,14 @@ test('cid correctly sets the id for the select and it\'s label', function(assert
   assert.equal(this.$('label').attr('for'), 'test-cid', 'label has correct \'for\'');
 });
 
+test('cid it property by default', function(assert) {
```
Comment (pedrokiefer) [inline tests/unit/components/em-select-test.js]: Same typo

File: tests/unit/components/em-text-test.js [hunk 4]
```diff
@@ -77,3 +77,13 @@ test('cid correctly sets the id for the textarea and it\'s label', function(asse
   assert.equal(component.$('textarea').attr('id'), cid, 'textarea has correct id');
   assert.equal(component.$('label').attr('for'), cid, 'label has correct \'for\'');
 });
+
+test('cid it property by default', function(assert) {
```
Comment (pedrokiefer) [inline tests/unit/components/em-text-test.js]: Yet again

**Comment by pedrokiefer** [comment]
@GCorbel did a small review of your patch, just some typos. :) 

Looks good to me!

File: addon/components/form-group.js (hunk 1 above)
Comment (GCorbel) [inline addon/components/form-group.js]: Hoops. Sorry for this mistake.

**Comment by GCorbel** [comment]
Done. Thanks @pedrokiefer for the review.

**Comment by spruce** [comment]
what happens if the property or the cid is not html compatible? Is that event possible? something like "mz cid is not valid ...".
Otherwise :+1:

**Comment by GCorbel** [comment]
@spruce I don't understand what you mean by "not html compatible". Will ember-rapid-forms works in those cases?

**Comment by spruce** [comment]
@GCorbel I mean when you put in a string like "not comforming" (with a space) which isn't allowed as an ID, right?

**Comment by GCorbel** [comment]
Do you think we have to manage this case? I think it will simply set the id with a space and I think it's normal.

**Comment by spruce** [comment]
Okay.
:+1:

**Pull request merged**